src/test/objectstore/TestRocksdbOptionParse.cc

```diff
@@ -38,8 +37,7 @@ TEST(RocksDBOption, simple) {
   ASSERT_EQ(104857600u, options.max_bytes_for_level_base);
   ASSERT_EQ(10485760u, options.target_file_size_base);
   ASSERT_EQ(3, options.num_levels);
-  ASSERT_FALSE(options.disableDataSync);
- // ASSERT_EQ("none", options.compression);
+  ASSERT_EQ(rocksdb::kNoCompression, options.compression);
```

Comment:
@tchaikov I guess `rocksdb::kNoCompression` is some kind of constant defined in `rocksdb/db.h`

Anyway, LGTM